### PR TITLE
JBIDE-27032 - Update Maven ITests to JavaEE 8/7 components

### PR DESCRIPTION
**Pull request by zcervink**
**JBIDE-27032 - Update Maven ITests to JavaEE 8/7 components**

**JIRA:** https://issues.redhat.com/browse/JBIDE-27032
**Jenkins:** https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/maven.itests/1013/

Signed-off-by: Zbynek Cervinka <zcervink@redhat.com>

Please review @odockal @jkopriva

### CODE REVIEW

File: tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/configurator/CDIConfiguratorTest.java [hunk 1]
```diff
@@ -30,8 +30,8 @@
 @JBossServer(state=ServerRequirementState.PRESENT)
 public class CDIConfiguratorTest extends AbstractConfiguratorsTest{
 	
-	public static final String CDI_API_VERSION1_1="1.1";
-	public static final String CDI_API_VERSION1_0="1.0-SP4";
+	public static final String CDI_API_VERSION_1_2="1.2";
+	public static final String CDI_API_VERSION_2_0="2.0";
 	public static final String SEAM_FACES_VERSION="3.0.0.Alpha3";
 	public static final String SEAM_INTERNATIONAL_VERSION="3.0.0.Alpha1";
 	public static final String DELTASPIKE_CORE_API_VERSION="0.3-incubating";
```
Comment (jkopriva) [inline tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/configurator/CDIConfiguratorTest.java]: What happen, if we use new version of deltaspike?

Comment (zcervink) [reply tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/configurator/CDIConfiguratorTest.java]: Fixed.

**Comment by zcervink** [comment]
@jkopriva All remaining updates has been performed and sonar is OK now.

**Comment by zcervink** [comment]
@jkopriva 
**Updated Jenkins is here:** https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/maven.itests/1022/

**Comment by jkopriva** [comment]
LGTM, merging.
@zcervink thanks for contribution, please update JIRAs related to test failures. Thank you!

**Pull request merged**